enhancements/network/static-ip-addresses-vsphere.md

```diff
@@ -60,6 +60,7 @@ As an OpenShift engineer, I want to know if a IPI installation was performed so
   Rationale: Some users must qualify each tool used in their environment. 
   Leveraging IPI greatly reduces the number of tools required to provision
   a cluster.
+- 
```

Comment:
what about UPI? Is it just a matter of creating VMs via vsphere api, so it's being done by the admin anyhow, or there is some configuration that propagates to OCP?

Reply:
The [UPI installation documentation for vSphere](https://docs.openshift.com/container-platform/4.11/installing/installing_vsphere/installing-vsphere-network-customizations.html#installation-vsphere-machines_installing-vsphere-network-customizations) covers installation of a UPI vSphere cluster with static IP addresses.  One piece of this enhancement that would still be applicable, though currently out of scope, would be support for scaling machines with static IPs.